packages/core/utils/src/config.js

```diff
@@ -71,7 +71,7 @@ export async function loadConfig(
 
     try {
       let extname = path.extname(configFile).slice(1);
-      if (extname === 'js') {
+      if (['js', 'cjs'].includes(extname)) {
```

Comment:
Could be slightly faster, avoids array allocation.

```suggestion
      if (extname === 'js' || extname === 'cjs') {
```

Reply:
Oh yes, good point